package.json

```diff
@@ -14,8 +14,10 @@
 		"dev:watch:modules": "tsc -w --declaration --sourceMap --outDir libs -p .",
 		"dev:watch:bundles": "rollup -w -c",
 		"dev:serve": "ws",
+		"storybook": "start-storybook -s ./public -p 6006",
 		"dev": "npm-run-all --parallel dev:watch:modules dev:watch:bundles dev:serve",
-		"prepublishOnly": "npm run clean && npm run build"
+		"prepublishOnly": "npm run clean && npm run build",
+		"build-storybook": "build-storybook"
```

Comment:
What is this script used for?

Reply:
It's part of storybook cli which can build static output, it's not being used at the moment. 

I'd like to in the future have storybook built as static for the gh-pages branch. That way people have a nice sandbox to explore the libs feature set.

Reply:
Sounds great. This process can be automated using GitHub Actions in the future.

Is dist/ directory also for this purpose? I think `npm run build-storybook` is no different from `npx build-storybook`, and we don't need them currently.

Reply:
I's say for the scope of this PR let's not worry about the built output of storybook.

Do you want me to remove this script?

Reply:
Yeah, plz remove it.